### DISCUSSION TITLE
Hue wall switch module Z2M

### DISCUSSION THREAD

**Discussion by lowrisk75** [Koenkk/zigbee2mqtt#13979] [Devices]
I just received two Hue wall switch module, the first one was paired immediately, but I can't pair the second one.
I'm not sure how to reset the switch since I can't find that in the docs, but I try to press the reset button a few time, a red link is blinking, I move the Hue wall switch module at a different location, but Z2M won't detect it.
any idea?

**Comment by SuneRadich** [comment]
Is it this one you mean https://www.zigbee2mqtt.io/devices/E1743.html ?

I recently bought a Sonoff ZBDongleE and cant get my Ikea switch paired correctly. Are you on the same stick?

**Reply by lowrisk75** [reply]
no I had the issue with this [Philips 929003017102](https://www.zigbee2mqtt.io/devices/929003017102.html)
The Ikea button you are referencing, I managed to add to Z2M but it's pointless, the battery die after a few hours/days
I do have issue with other devices like [IKEA E1525/E1745](https://www.zigbee2mqtt.io/devices/E1525_E1745.html).

I couldn't pair a [Philips 8718699693985](https://www.zigbee2mqtt.io/devices/8718699693985.html) but I just updated the firmware of my Sonoff Zigbee 3.0 USB, and I managed ti pair it afterward (unfortunately still the same problems with the other devices)
